packages/timeline-state-resolver-types/src/integrations/atem.ts

```diff
@@ -414,3 +416,13 @@ export interface TimelineContentAtemAudioRouting extends TimelineContentAtemBase
 		sourceId: number
 	}
 }
+
+export interface TimelineContentAtemColorGenerator extends TimelineContentAtemBase {
+	type: TimelineContentTypeAtem.COLORGENERATOR
+
+	colorGenerator: {
+		hue: number
+		saturation: number
+		luma: number
```

Comment:
Minor note: Could you add descriptions for these, detailing what number ranges are expected?